src/program/lwaftr/check/util.lua

```diff
@@ -18,13 +15,24 @@ function load_requested_counters(counters)
    return result
 end
 
+local function counter_names()
+   local names = {}
+   local schema = schema.load_schema_by_name('snabb-softwire-v2')
+   for k, node in pairs(schema.body['softwire-state'].body) do
```

Comment:
What do you think about taking advantage of this PR to put counters into their own container? I would need to adapt this code when alarms get added. I can live without it for now though.